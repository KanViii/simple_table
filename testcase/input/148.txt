INSERT a number
BEGIN
INSERT a string
ASSIGN a 'hello'
LOOKUP a
BEGIN
INSERT b number
ASSIGN b 123
LOOKUP b
END
END
LOOKUP a