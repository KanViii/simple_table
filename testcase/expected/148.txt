success
success
success
1
success
success
2
0